src/jit/jitconfigvalues.h

```diff
@@ -200,8 +200,13 @@ CONFIG_INTEGER(AltJitAssertOnNYI, W("AltJitAssertOnNYI"), 0) // Controls the Alt
 CONFIG_INTEGER(AltJitAssertOnNYI, W("AltJitAssertOnNYI"), 1) // Controls the AltJit behavior of NYI stuff
 #endif                                                       // defined(_TARGET_ARM64_) || defined(_TARGET_X86_)
 
+#if defined(_TARGET_X86_) || defined(_TARGET_AMD64_)
+CONFIG_INTEGER(EnableSSE3_4, W("EnableSSE3_4"), 1) // Enable SSE3, SSSE3, SSE 4.1 and 4.2 instruction set as default
+#endif
+
 #if defined(_TARGET_AMD64_)
-CONFIG_INTEGER(EnableAVX, W("EnableAVX"), 1) // Enable AVX instruction set for wide operations as default
+CONFIG_INTEGER(EnableAVX, W("EnableAVX"), 1) // Enable AVX instruction set for wide operations as default.
+                                             // If AVX is enabled, then it will take precendence over EnableSSE4.
 #else                                        // !defined(_TARGET_AMD64_)
```

Comment:
It is valid to have both being set or unset.

If both are set, then JIT will use the highest instruction set available.
If both are unset, then JIT will use SSE2.

I will add comments to clarify this here.

---
In reply to: [89866902](https://github.com/dotnet/coreclr/pull/8291#discussion_r89866902) [](ancestors = 89866902,89866493)